epics_det.py

```diff
@@ -196,7 +196,7 @@ def det_waitArmed(): #not even sure what this means.
     start_time = time.time()
     while (get_det_pv("armed_state") == 0):
       if time.time() - start_time > timeout:
-        logger.info('aborting arm')
+        logger.error('aborting arm')
```

Comment:
Maybe add "Timed out" or some such?

Reply:
agree with this - fixed with commit https://github.com/NSLS-II/lsdc/pull/104/commits/a98e6cfa78b50173f1b72839359844628d3090ec